### ISSUE TITLE
Adding third-party library results in white-screen

### ISSUE THREAD

**Issue by cbrevik** [ds300/react-native-typescript-transformer#5]
Adding `bugsnag-react-native` and importing it in file results in white-screen in app. 

When stepping through code in debugger, it is still calling the relevant `render()` functions of components, so not sure why it behaves like this.

Reproducible code [here](https://github.com/cbrevik/tsWithTransformer/tree/14b53dfbe66aecd0b60ed32f3d9e5dc4030694fc).

**Comment by ds300** [comment]
Again, that commit works fine for me on iOS and Android. Did clearing the cache help with this too?

**Comment by cbrevik** [comment]
Of course it did, sorry for wasting your time. 😞 

Find it a bit weird that I have to do that for both issues though. I didn't do anything special I wouldn't have done otherwise while setting up a project.

**Comment by cbrevik** [comment]
Think I might have uncovered why I got this problem.

I'm having an issue with VSCode not finding the `bugsnag-react-native` module with intellisense:
![image](https://cloud.githubusercontent.com/assets/4932625/26026044/aabf8aaa-37f4-11e7-84a2-f8e2182c88a6.png)
![image](https://cloud.githubusercontent.com/assets/4932625/26026052/c3202546-37f4-11e7-97c2-1f301efcb91f.png)

I tried changing to `"module": "commonjs"` in tsconfig, which solved the intellisense. That seems also to have caused the white screen though. Just re-running normally then with "module": "es2015" wasn't enough.
